apps/openmw/mwworld/inventorystore.cpp

```diff
@@ -266,6 +266,7 @@ void MWWorld::InventoryStore::autoEquip (const MWWorld::Ptr& actor)
     // Autoequip clothing, armor and weapons.
     // Equipping lights is handled in Actors::updateEquippedLight based on environment light.
 
+    // the main loop iterating through all items in inventory
```

Comment:
Please don't litter the code with redundant comments. Comments that only describe what the code does are not helpful you might as well just read the code.